code/modules/mob/living/carbon/carbon.dm

```diff
@@ -202,7 +202,8 @@
 		visible_message("<span class='danger'>[src] has thrown [thrown_thing].</span>")
 		log_message("has thrown [thrown_thing]", LOG_ATTACK)
 		do_attack_animation(target, no_effect = 1)
-		playsound(loc, 'sound/weapons/punchmiss.ogg', 50, 1, -1)
+		if(throwforce)
+			playsound(loc, 'sound/weapons/punchmiss.ogg', 50, TRUE, -1)
```

Comment:
Don't forget to mark it as a Skyrat change if it isn't in modular. 
```suggestion
		if(throwforce) //Skyrat change
			playsound(loc, 'sound/weapons/punchmiss.ogg', 50, TRUE, -1) //Skyrat change
```

Reply:
Added tags on begin and end, which should be hopefully a bit more visible.